browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -382,6 +382,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			spinfield.setAttribute('disabled', 'true');
 		}
 
+		var enabled = Boolean(data.enabled);
```

Comment:
In case of missing `enabled` property it will be `undefined` (it is optional by default it is enabled).
Then `!enabled` will be `true` and will disable widget which is not good.

Please use explicit check for `const disabled = data.enabled === false;`

Reply:
We use that just above in the if condition. Would be good to have single variable and reuse it.

Reply:
@hcvcastro could you please adjust code to above corner-case?
apart of that is ready to merge